docs/pages/develop/user-interface/fonts.mdx

```diff
@@ -92,7 +92,7 @@ Add the config plugin to your [app config](/versions/latest/config/app/#plugins)
 
 <Step label="3">
 
-After embedding the font with the config plugin, create a [new development build](/develop/development-builds/create-a-build/) and install it on your device or Android Emulator or iOS Simulator.
+Run `npx expo prebuild --clean` to apply the config plugin. Start a new build of the app (`npx expo run:[android|ios]`) &mdash; the font files are embedded into the app binary during build. Install the app on your device or Android Emulator or iOS Simulator.
```

Comment:
At this stage, the user is unaware of what prebuild is in the user journey. All guides in the "Home" section use a recommended path, which is to create a development build. The text before is accurate in the previous context. 

Let's not add local compilation and prebuild command in this step and the step 3 of Google fonts since it leaves room for confusion.

Reply:
I reverted this